.github/workflows/main.yml

```diff
@@ -0,0 +1,83 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  jjb:
+    runs-on: ubuntu-latest
+    name: Jenkins Job Builder
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Restore pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          echo 'jenkins-job-builder' >> requirements.txt
+          echo 'lxml' >> requirements.txt
+          echo 'requests' >> requirements.txt
```

Comment:
Can't we store `requirements.txt` in git as a file?

Reply:
We could. @evgeni suggested this method, y'all figure it out and tell me what you want.

Reply:
I did not, I meant to have the file in git. ;)